app/views/articles/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title @article.title %>
 
 <%= render "shared/webcomponents_loader_script" %>
-<%= javascript_packs_with_chunks_tag "clipboardCopy", "webShare", "articlePage", defer: true %>
+<%= javascript_packs_with_chunks_tag "clipboardCopy", "webShare", "articlePage", "loadActionsPanelButton", defer: true %>
```

Comment:
Consider renaming the pack to `articleModerationTools` or something more descriptive.

Reply:
done! 👍